### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by ZoicanDenisUi**
Created the necessary files for the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            <input type="number" id="ballCountInput">
+        </div>
+        <button id="startButton">Bounce</button>
+        
+    </div>
+
+    <div id="bounceDiv" ></div>
+    
+</body>
+<script src='main.js'></script>
```
Comment (sergiubologa) [inline index.html]: Why is this outside of the body tag?

**Review by sergiubologa — CHANGES_REQUESTED** [review]
Please add a description to each PR with short info about what that PR wants to accomplish.

File: index.html (hunk 1 above)
Comment (ZoicanDenisUi) [reply index.html]: I moved the script tag inside the body tag.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    console.log(newDiv.style.top)
+    bounceDiv.append(newDiv)
+
+});
+
+
+
+// Animation 
+  var bounceAnimation = [
```
Comment (ioanajoj) [inline main.js]: Try to always go for `const`, when needed use `let`, and hopefully never `var`

File: main.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    let diffRightLeft = rightBounceDiv-leftBounceDiv
+    let difTopBottom = bottomBounceDiv - topBounceDiv
+
+    const newDiv = document.createElement("div");
+    const circleRadius = 40
+    newDiv.style.width=circleRadius+"px"
+    newDiv.style.height=circleRadius+"px"
+    newDiv.style.backgroundColor="red"
+    newDiv.style.borderRadius="100%"
+    newDiv.style.position="absolute"
```
Comment (ioanajoj) [inline main.js]: I would extract the styles that are not 'configurable' into a css class and assign it to newDiv

**Review by ioanajoj — CHANGES_REQUESTED** [review]

**Review by sergiubologa — APPROVED** [review]

**Review by ioanajoj — APPROVED** [review]

**Pull request merged**